Gems/Atom/Feature/Common/Code/Source/ReflectionProbe/ReflectionProbe.cpp

```diff
@@ -138,43 +138,39 @@ namespace AZ
             if (m_updateSrg)
             {
                 // stencil Srg
-                // Note: the stencil pass uses a slightly reduced inner AABB to avoid seams
+                // Note: the stencil pass uses a slightly reduced inner OBB to avoid seams
                 Vector3 innerExtentsReduced = m_innerExtents - Vector3(0.1f, 0.1f, 0.1f);
-                Matrix3x4 modelToWorldStencil = Matrix3x4::CreateFromMatrix3x3AndTranslation(Matrix3x3::CreateIdentity(), m_transform.GetTranslation()) * Matrix3x4::CreateScale(innerExtentsReduced);
+                Matrix3x4 modelToWorldStencil = Matrix3x4::CreateFromQuaternionAndTranslation(m_transform.GetRotation(), m_transform.GetTranslation()) * Matrix3x4::CreateScale(innerExtentsReduced);
```

Comment:
Just `Matrix3x4::CreateFromTransform(m_transform) * Matrix3x4::CreateScale(innerExtentsReduced)` would work here and elsewhere in this file I think. It'd be faster to apply the scale on the transform directly also (copy xform if you want to preserve the original).